TransferMethod/Tests/AddTransferMethodPresenterTests.swift

```diff
@@ -36,6 +39,20 @@ class AddTransferMethodPresenterTests: XCTestCase {
         XCTAssertTrue(mockView.isShowLoadingPerformed, "The showLoading should be performed")
         XCTAssertTrue(mockView.isHideLoadingPerformed, "The hideLoading should be performed")
         XCTAssertEqual(mockView.fieldGroups.count, 2, "The `response.getFields()` should be 2")
+
+        XCTAssert(mockHyperwalletInsights.trackImpression, "HyperwalletInsights.trackImpression should be called")
+        XCTAssert(mockHyperwalletInsights.pageName == "transfer-method:add:collect-transfer-method-information",
+                  "Page name should be transfer-method:add:collect-transfer-method-information")
+        XCTAssert(mockHyperwalletInsights.pageGroup == "transfer-method",
+                  "Page group should be transfer-method")
+        XCTAssert((mockHyperwalletInsights.params[InsightsTags.country] != nil),
+                  "Params should have country")
+        XCTAssert((mockHyperwalletInsights.params[InsightsTags.currency] != nil),
+                  "Params should have currency")
+        XCTAssert((mockHyperwalletInsights.params[InsightsTags.profileType] != nil),
+                  "Params should have profile type")
+        XCTAssert((mockHyperwalletInsights.params[InsightsTags.transferMethodType] != nil),
+                  "Params should have transfer method type")
```

Comment:
Use XCTAssertNotNil instead of checking nil

Reply:
It has been updated based #165 PR changes.